### ISSUE TITLE
Fix / improve information about rendered docs (paths) that is shown in output

### ISSUE THREAD

**Issue by sypets** [t3docs/docker-render-documentation#11]
dockrun_t3rdf makehtml currently produces the output:

```
Find the (possible) results. For example:
   ./Documentation-GENERATED-temp/.../Index.html
   ./Documentation-GENERATED-temp/.../singlehtml/Index.html
   ./Documentation-GENERATED-temp/.../_pdf/
   ./Documentation-GENERATED-temp/.../_buildinfo/
   ./Documentation-GENERATED-temp/.../_buildinfo/warnings.txt
```
* [ ] It might be helpful to show entire path at beginning
* [ ] Remove paths that are not generated from this information: singehtml, _pdf

**Comment by sypets** [comment]
I would vote to close this for now. If you want to work on it, you can reopen.

**Comment by sypets** [comment]
I still get the output with the 3 dots. 

```
Final exit status: 0 (completed)

Find the (possible) results. For example:
   ./Documentation-GENERATED-temp/.../Index.html
   ./Documentation-GENERATED-temp/.../singlehtml/Index.html
   ./Documentation-GENERATED-temp/.../_pdf/
   ./Documentation-GENERATED-temp/.../_buildinfo/
   ./Documentation-GENERATED-temp/.../_buildinfo/warnings.txt

```

I would prefer to get full relative or full complete path, for example:

**Documentation-GENERATED-temp/Result/project/0.0.0/Index.html**

I initially suggested full path. My idea was to get an output with a full URL you can click on in the terminal, for example:

echo "file://"$(pwd)"/Documentation-GENERATED-temp/Result/project/0.0.0/Index.html"

This would result in something like this in the terminal: 

file:///home/peters/prog.local/github-pull-requests/typo3-documentation/TYPO3CMS-Guide-HowToDocument/Documentation-GENERATED-temp/Result/project/0.0.0/Index.html

Depending on the type of terminal, you might be able to click on it and let it open in the browser.

This would make opening rendered start page in the browser even easier. 

If you look in **Documentation-GENERATED-temp/last-docker-run-command-GENERATED.sh**: this file was created during rendering and does contain the complete path on host. So this information must be available in the Docker containter: 

Documentation-GENERATED-temp/last-docker-run-command-GENERATED.sh:

```
docker run --rm --user=1000:1000 \
   -v /home/peters/prog.local/github-pull-requests/typo3-documentation/TYPO3CMS-Guide-HowToDocument:/PROJECT:ro \
   -v /home/peters/prog.local/github-pull-requests/typo3-documentation/TYPO3CMS-Guide-HowToDocument/Documentation-GENERATED-temp:/RESULT t3docs/render-documentation:v1.6.10-full makehtml

```

**Comment by marble** [comment]
Fixed in master >= v2.0.0
Example:
![grafik](https://user-images.githubusercontent.com/307057/58379857-80e1e800-7fa9-11e9-9195-fe540e2c02c2.png)
